scripts/search/tests/data/dataset-finch1.jsonld

```diff
@@ -0,0 +1,26 @@
+
+{
+  "http://purl.org/dc/terms/title": "Darwin's Finches",
+  "http://purl.org/dc/terms/subject": "Medicine, Health and Life Sciences",
+  "http://purl.org/dc/terms/creator": {
+      "https://dataverse.org/schema/citation/author#Name": "Finch, Fiona",
+      "https://dataverse.org/schema/citation/author#Affiliation": "Birds Inc."
+  },
```

Comment:
How do I specify multiple authors?

Reply:
An array of objects with the name/affiliation keys as the value of the creator key. This is the same way the OAI-ORE export shows multiple authors.

Reply:
Having a human-readable term label (that probably isn't globally unique/machine interpretable) while still supporting strict machine readability adds some complexity, but its pretty straight forward to either never use an @context or to always use a standard/static one.

Regardless, it wouldn't be hard to add documentation over time.